h/templates/client/annotation.html

```diff
@@ -97,6 +97,13 @@
 </div>
 <!-- / Tags -->
 
+<div class="annotation-section small" ng-if="vm.editing">
+  <p ng-show="level.text == 'Only Me'">
```

Comment:
You can just use `vm.isPrivate()` here and then you don't need `setLevel` and you don't need to reach "up" out of the isolated privacy directive scope (which is naughty thing to do).